### DISCUSSION TITLE
icon svg

### DISCUSSION THREAD

**Discussion by runsense** [primefaces/community#2528] [PrimeReact] [answered]
hello,
i try to insert svg for
Items in TieredMenu .

found in doc icon type accept
SVGProps<SVGSVGElement>

so i try to pass svg inside, 
function like in exemple of
Dropdown 

but no way to insert it
like state work.

existe t'il une manière de
Merci

**Comment by ShiYaoDiving** [comment]
There is a video about that.
And i use the icomoon~It can works just like primeicon

**Reply by melloware** [reply]
see: https://primereact.org/customicons/#svg

**Comment by runsense** [comment, marked as answer]
dropdown icon am sure work like this but for another menu
icon property have no option to call.
but found a way with span inside and svg!
